### PR TITLE
Starting instructor notes

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

**Review by jcohen02 — APPROVED** [review]
This looks like a great start to the instructor notes. All really important points and things that seem familiar from a couple of experiences of running courses using this material.

I think there are a few additional points to add around confusion from Linux users (maybe this affects Mac too?) about when and why to use `sudo` and then about file ownership, especially in files created in bind mounted directories (although actually it looks like this might now be much less of an issue from recent experience).

The points made above should probably go into a separate issue which I'll open - as a start for the instructor notes, this looks great.

File: _extras/guide.md [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+installations.
+* **Install Issues**: Windows computers seem to consistently have issues with Docker.
+Consider having people check their install in advance at a separate time or come early.
+In online workshops, consider using breakout rooms to have smaller groups for
+participants to demonstrate their installations. Use a more complex command than
+`docker --version` to test the installation.
+* **Virtualization Illustration**: When going through the intro to containers,
+consider demonstrating what this might look like by having two shells (or shell tabs)
+open, one on your host computer and one into a container you started before the
+workshop. Then you can demonstrate in a simple way that from the same (host) computer,
```
Comment (sstevens2) [inline _extras/guide.md:20]: Maybe add in a suggested example command like `whoami` or `pwd`?

**Review by sstevens2 — APPROVED** [review]
I think it looks great!  My small suggest is that maybe add a couple example commands that you might use to show the different envs.

File: _extras/guide.md (hunk 1 above)
Comment (ChristinaLK) [reply _extras/guide.md:20]: Good idea! I tried to do a sub-list with ideas, am not sure the formatting will work.

**Pull request merged**